### PR TITLE
Add PERP to the token list

### PR DESCRIPTION
**Pull request by Alvinhung**

### CODE REVIEW

File: tags/tags.json [hunk 1]
```diff
@@ -77,5 +77,10 @@
     "defi": {
         "name": "Defi",
         "description": null
+    },
+    "perpetuals": {
+        "name": "Perpetuals",
+        "description": "Tokens that trustlessly recreate decentralized perpetual contracts on other assets",
+        "emoji":"🔄"
```
Comment (dharmabrendan) [inline tags/tags.json:84]: We considered this tag, but do not think that it adds value at this time. Please delete this addition

File: token-metadata/0xbC396689893D065F41bc2C6EcbeE5e0085233447/metadata.json [hunk 2]
```diff
@@ -4,5 +4,23 @@
     "address": "0xbC396689893D065F41bc2C6EcbeE5e0085233447",
     "decimals": 18,
     "dharmaVerificationStatus": "VERIFIED",
-    "tags": ["yield-farming"]
+    "tags": ["synthetics","amm","defi","perpetuals"],
```
Comment (dharmabrendan) [inline token-metadata/0xbC396689893D065F41bc2C6EcbeE5e0085233447/metadata.json:7]: per comment above, please remove `perpetuals` from the tags list

**Review by dharmabrendan — COMMENTED** [review]
Overall looks great, please address the comments and then we'll merge in

File: token-metadata/0xbC396689893D065F41bc2C6EcbeE5e0085233447/metadata.json (hunk 2 above)
Comment (Alvinhung) [reply token-metadata/0xbC396689893D065F41bc2C6EcbeE5e0085233447/metadata.json:7]: okay, thanks for the correction

**Comment by dharmabrendan** [comment]
Thanks for your submission @Alvinhung! We have approved and merged in :)

**Comment by Alvinhung** [comment]
Thanks for your hard work! Keep it up.

**Pull request merged**